codegen-sbt/src/sbt-test/codegen/test-compile/test

```diff
@@ -1,14 +1,13 @@
-$ exists project/schema.graphql
-$ absent src/main/scala/Client.scala
 $ absent src/main/scala/client/ClientNameUniqueness.scala
+$ absent src/main/scala/client/ClientGitLab.scala
 $ mkdir src/main/scala
 $ mkdir src/main/scala/client
 
-> calibanGenClient project/schema.graphql src/main/scala/Client.scala
-$ exists src/main/scala/Client.scala
 > calibanGenClient project/schema-to-check-name-uniqueness.graphql src/main/scala/client/ClientNameUniqueness.scala --packageName client
 $ exists src/main/scala/client/ClientNameUniqueness.scala
 > calibanGenClient project/gitlab-schema.graphql src/main/scala/client/ClientGitLab.scala --packageName client
 $ exists src/main/scala/client/ClientGitLab.scala
 
 > compile
+
+$ exists target/scala-2.12/src_managed/main/caliban-codegen-sbt/caliban/Client.scala
```

Comment:
To make scripted tests faster, how about using ` --genView true` in `test-compile` and getting rid of `gen-view-test-compile`? I don't think we need both, as `--genView` is just adding more code.